common/src/error.rs

```diff
@@ -6,5 +6,9 @@ pub enum DenimBufferError {
     ChunkEncodeError,
     MinPayloadLengthTooHighError,
     ChunkBufferNotFound,
-    NoChunksInDeniablePayloadError,
+}
+
+#[derive(Debug, Display, Error, From)]
+pub enum DenimEncodeError {
```

Comment:
Should this be an enum? are there any other types of encode errors?

Reply:
Før var det bare en BufferError og det har intet med Bufferen at gøre